src/udp_packets.rs

```diff
@@ -0,0 +1,277 @@
+pub mod udp_payload {
+    // const MAX_PAYLOAD_IPV4: usize = 65507; // 65,535 - 20 (IP header) - 8 (UDP header)
+    // const MAX_PAYLOAD_IPV6: usize = 65527; // 65,535 - 40 (IPv6 header) - 8 (UDP header)
+
+    /// From RFC 768
```

Comment:
question: worth a link to the actual RFC?

i.e. `/// From [RFC 768](https://www.rfc-editor.org/rfc/rfc768.txt)`

Reply:
These links were a suggestion from @bee-san, which I may have misinterpreted. For now, this can all be removed, as we're sending blank packets anyway (from what I understand)

Reply:
Oh, I'm not suggesting removing these docs. (they're a great addition) just citing the original for further reading.